### ISSUE TITLE
Make unmuted icon red

### ISSUE THREAD

**Issue by semente** [wbolster/nothing-to-say#5]
Just to call your attention to you not forget it on.

**Comment by wbolster** [comment]
yes, good idea. not sure how though since this extensions uses the default icons.

**Comment by YodaDaCoda** [comment]
I'm not sure that making it red is a great idea. The only red icons I have are indicating an error status; this would be confusing to me personally. I can respect the option of having it configurable however.

**Comment by wbolster** [comment]
no intention to change this for the reason @YodaDaCoda mentioned
